components/Navigation/DesktopNav/DesktopNav.tsx

```diff
@@ -20,6 +21,13 @@ const DesktopNavItem = ({ link }: DesktopNavItemProps) => (
                     </span>
                 </a>
             </Link>
+        ) : link.isExternalLink ? (
```

Comment:
Works well, I like the support for controlling external link icons via the NAVIGATION_LINKS constant instead of hardcoding an exception